container_workflow_tool/distgit.py

```diff
@@ -241,10 +256,15 @@ def dist_git_changes(self, images, rebase=False):
                 pull_upstr = image.get("pull_upstream", True)
                 repo = self._clone_downstream(component, branch)
                 df_path = os.path.join(component, "Dockerfile")
+                self.logger.info(df_path)
```

Comment:
This seems like debugging call. Either remove or change it into a call to `.debug()` with more information please.

Reply:
I will remove it. Sorry, I have forgot to delete it.